snuba/migrations/runner.py

```diff
@@ -136,14 +136,25 @@ def get_status(migration_key: MigrationKey) -> Status:
 
         return migrations
 
-    def run_all(self, *, force: bool = False) -> None:
+    def run_all(self, *, force: bool = False, group: Optional[str] = None) -> None:
         """
-        Run all pending migrations. Throws an error if any migration is in progress.
+        If group is specified, runs all pending migrations for that specific group. Makes
+        sure to run any pending system migrations first so that the migrations table is
+        created before running migrations for other groups. Throw an error if any migration
+        is in progress.
+
+        If no group is specified, run all pending migrations. Throws an error if any
+        migration is in progress.
```

Comment:
nit: this line is written twice, once is enough: `Throws an error if any migration is in progress`